README.md

```diff
@@ -287,7 +287,7 @@ The purpose of sample filtering at this stage after EvidenceQC is to prevent ver
 
 
 ## <a name="gcnv-training">TrainGCNV</a>
-Trains a gCNV model for use in [GatherBatchEvidence](#gather-batch-evidence). The WDL can be found at `/wdl/TrainGCNV.wdl`. See the [gCNV training overview](#gcnv-training-overview) for more information.
+Trains a [gCNV](https://gatk.broadinstitute.org/hc/en-us/articles/360035531152) model for use in [GatherBatchEvidence](#gather-batch-evidence). The WDL can be found at `/wdl/TrainGCNV.wdl`. See the [gCNV training overview](#gcnv-training-overview) for more information.
```

Comment:
Can you also link to gCNV under "[gCNV training](https://github.com/broadinstitute/gatk-sv/tree/main#gcnv-training-overview)" (line 200) since that appears earlier in the README? Sorry if this was confusing in my last comment since there are two sections on gCNV training. It's fine to either leave this link or remove it once you add the other one, since the two sections link to each other